jbi/services/jira.py

```diff
@@ -30,15 +31,45 @@
 instrumented_method = instrument(prefix="jira", exceptions=(errors.ApiError,))
```

Comment:
This is a drive by fix, but I noticed when investigating the `log_http_error` decorator that the Jira client doesn't ever raise `errors.ApiError` -- it only raises `request.HTTPError`s. Should we change this line to reflect that?

Reply:
OH, but that's probably why we have #345 !

Reply:
Nice! Good catch.